cli/src/lib/bootstrapShell.js

```diff
@@ -66,14 +66,17 @@ const bootstrapShell = async ({ paths, shell, force = false }) => {
 
     const srcNodeModules = path.join(source, 'node_modules')
     const destNodeModules = path.join(dest, 'node_modules')
+    reporter.debug(
+        `Bootstrapping appShell. Source: "${srcNodeModules}" - Destination: "${destNodeModules}"`
+    )
     if (fs.exists(srcNodeModules)) {
         reporter.debug(
             `Linking ${path.relative(
                 paths.base,
                 destNodeModules
             )} to ${path.relative(paths.base, srcNodeModules)}...`
         )
-        await fs.ensureSymlink(srcNodeModules, destNodeModules)
+        fs.ensureSymlinkSync(srcNodeModules, destNodeModules)
```

Comment:
this - and other fs.sync calls - fix a race condition that happened before the first time we run `start` where the symlink is sometimes not yet created